rds-deployment-HA/azuredeploy.json

```diff
@@ -0,0 +1,397 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique public DNS prefix for the deployment. The fqdn will look something like '<dnsname>.westus.cloudapp.azure.com'. Up to 62 chars, digits or dashes, lowercase, should start with a letter: must conform to '^[a-z][a-z0-9-]{1,61}[a-z0-9]$'."
+      }
+    },
+    "loadBalancer": {
+      "type": "string",
+      "metadata": {
+        "description": "The Loadbalancer name must match from the RDS deployment. And default value taken by template is loadbalancer."
+      },
+      "defaultValue": "loadBalancer"
+    },
+    "backendAddressPools": {
+      "type": "string",
+      "defaultValue": "LBBAP",
+      "metadata": {
+        "description": "The backendAddressPools name must match from the RDS deployment. And default value taken by template is LBBAP."
+      }
+    },
+    "gw-availabilityset": {
+      "type": "string",
+      "defaultValue": "gw-availabilityset",
+      "metadata": {
+        "description": "The gw-availabilityset name must match from the RDS deployment. And default value taken by template is gw-availabilityset."
+      }
+    },
+    "adDomainName": {
+      "type": "string",
+      "metadata": {
+        "description": "The FQDN of the AD domain"
+      }
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the administrator of the new VM and the domain. Exclusion list: 'administrator"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The password for the administrator account of the new VM and the domain"
+      }
+    },
+    "numberOfWebGwInstances": {
+      "type": "int",
+      "defaultValue": 2,
+      "metadata": {
+        "description": "Number of RDSH instances"
+      }
+    },
+    "BrokerServer": {
+      "type": "string",
+      "metadata": {
+        "description": "FQDN for Broker Server"
+      }
+    },
+    "WebURL": {
```

Comment:
We have updated the description comment for this. WebURL is simple public url name for RD gateway external FQDN.